src/types/mod.rs

```diff
@@ -80,6 +81,8 @@ pub(crate) mod date;
 pub(crate) mod error;
 
 pub(crate) mod internal;
+#[cfg(all(feature = "napi-1", feature = "symbol-primitive-api"))]
+pub(crate) mod symbol;
```

Comment:
Can you add a `use symbol::Symbol`? Currently it's not part of the `types` module.